components/FormField/src/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { HTMLAttributes } from 'react';
+import './index.scss';
+import clsx from 'clsx';
+
+export interface FormFieldProps extends HTMLAttributes<HTMLDivElement> {
+  label: string;
+  input: React.ReactNode;
```

Comment:
I think this should be `React.ReactElement`? ReactNode is throwing errors in my IDE for not defining some of the properties you're accessing below